### PR TITLE
Feature/fastqc

### PR DESCRIPTION
**Pull request by matthdsm**
Add fastqc.

This might require some more plumbing and/or configuring down the line, but we can add that once we have all the essential building blocks.

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [x] If you've fixed a bug or added code that should be tested, add tests!
 - [x] If necessary, also make a PR on the [nf-core/demultiplex branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/demultiplex)
 - [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [x] Make sure your code lints (`nf-core lint .`).
 - [ ] Documentation in `docs` is updated
 - [x] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/demultiplex/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: workflows/demultiplex.nf [hunk 1]
```diff
@@ -121,7 +123,12 @@ workflow DEMULTIPLEX {
...
     // MODULE: fastp
     FASTP(ch_parsed_fastq, [], [])
-    ch_fastp_multiqc = FASTP.out.json.map { meta, json -> return json}
+    ch_multiqc_files = ch_multiqc_files.mix( FASTP.out.json.map { meta, json -> return json} )
+    ch_versions = ch_versions.mix(FASTP.out.versions)
+
+    // MODULE: fastqc
+    FASTQC(ch_parsed_fastq)
+    ch_multiqc_files = ch_multiqc_files.mix( FASTQC.out.zip.map { meta, zip -> return zip} )
     ch_versions = ch_versions.mix(FASTP.out.versions)
```
Comment (jfy133) [inline workflows/demultiplex.nf:132]:
```suggestion
    ch_versions = ch_versions.mix(FASTQC.out.versions)
```

**Review by jfy133 — APPROVED** [review]
One correction but otherwise :+1:

File: workflows/demultiplex.nf (hunk 1 above)
Comment (matthdsm) [reply workflows/demultiplex.nf:132]: copy pasted too fast!

**Comment by matthdsm** [comment]
No really sure what I have to do to pass the remaining test, so I'm able to merge.

**Comment by jfy133** [comment]
Ah that's a faulty CI, I'll fix that for you

**Comment by jfy133** [comment]
There you go :)

**Comment by matthdsm** [comment]
Nice, thanks a lot!

**Pull request merged**